src/app/view/perspectives/chart/chart-perspective.component.ts

```diff
@@ -146,7 +146,7 @@ export class ChartPerspectiveComponent implements OnInit, OnDestroy {
   private subscribeData() {
     this.documents$ = this.store$.pipe(
       select(selectDocumentsByQuery),
-      distinctUntilChanged((x, y) => JSON.stringify(x) === JSON.stringify(y))
+      distinctUntilChanged((x, y) => deepObjectsEquals(x, y))
     );
```

Comment:
You could now use isEqual() from lodash (which also has isDeepEquals).

Reply:
`deepObjectsEquals` uses `isEqual()` from lodash. You did it recently ;)

Reply:
I see, it is the other way around!